src/encoding.ts

```diff
@@ -11,6 +11,16 @@ export function countRetinal(encoding: Encoding) {
   return count;
 }
 
+export function channels(encoding: Encoding) {
+  var arr = [];
+  CHANNELS.forEach(function(channel) {
```

Comment:
Want to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter ?